napari/utils/_register.py

```diff
@@ -31,10 +36,14 @@ def create_func(cls, name=None, doc=None, filename: str = '<string>'):
     name = 'add_' + name
 
     if doc is None:
+        # While the original class may have Attributes in its docstring, the
+        # generated function should not have an Attributes section.
+        # See https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes
         doc = getdoc(cls)
-        cutoff = doc.find('\n\nParameters\n----------\n')
-        if cutoff > 0:
-            doc = doc[cutoff:]
+        start = doc.find('\n\nParameters\n----------\n')
+        end = doc.find('\n\nAttributes\n----------\n')
+        if start > 0:
+            doc = doc[start:end]
```

Comment:
```
  E                     - dask arrays.
  E                     ?            -
  E                     + dask arrays
```

https://github.com/napari/napari/blob/3902911e622be29104273f5acfaf5ef253fbd069/napari/layers/tracks/tracks.py#L89-L91

The source of problem is that [`str.find](https://docs.python.org/3/library/stdtypes.html#str.find) returns -1 if do not find string, and there is no section `Attributes` for Track layer. 

```suggestion
        start = doc.find('\n\nParameters\n----------\n')
        end = doc.find('\n\nAttributes\n----------\n')
        if end == -1:
            end = None
        if start > 0:
            doc = doc[start:end]
```

Or add `Attributes` section to Tracks layer docstring.

Reply:
Ah! Thanks 👍🏻